modules/rules_sh/0.3.0/MODULE.bazel

```
@@ -0,0 +1,11 @@
+module(
+    name = "rules_sh",
+    version = "0.3.0",
+    compatibility_level = 0,
+    toolchains_to_register = ["@local_posix_config//:local_posix_toolchain"],
+)
+bazel_dep(name = "bazel_skylib", version = "1.0.3")
+bazel_dep(name = "platforms", version = "0.0.4")
+
+sh_configure = use_extension("@rules_sh//bzlmod:extensions.bzl", "sh_configure")
```

Just FYI, you can do:
```suggestion
sh_configure = use_extension("//bzlmod:extensions.bzl", "sh_configure")
```